internal/host/validator.go

```diff
@@ -957,6 +957,33 @@ func comparisonBuilder(value float64) string {
 	return equals
 }
 
+func (v *validator) generatePacketLossAdvisoryMessageForHost(c *validationContext, hostPacketLoss [] string) string {
+	message := "Actions:\n"
+	message += "1: Check if there are multiple devices on the same L2 network, if so then create a bond or disable all but one of the interfaces.\n"
+
+	var inventory models.Inventory
+	err := json.Unmarshal([]byte(c.host.Inventory), &inventory)
```

Comment:
There's a deserialized inventory cache Somewhere:tm: in this file, could it be used here?